src/clarity-icons/shapes/basic-shapes.ts

```diff
@@ -4,7 +4,8 @@
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-export const BasicShapes: any = {
+/* tslint:disable:max-line-length */
+export const basicShapes: any = {
```

Comment:
It was initially `essential`, `basic`,`social`, and `technology`. But we decided to change it to `core`, `basic`, `social`, and `technology`.